src/App.tsx

```diff
@@ -1,7 +1,43 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (this.getEventOfGlobalHandler));
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', (this.getEventOfGlobalHandler));
+  }
+
+  getEventOfGlobalHandler = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `${event.key}` });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return pressedKey.length
+      ? (
+        <div className="App">
+          <p className="App__message">
+            {`The last pressed key is [${pressedKey}]`}
+          </p>
+        </div>
+      )
+      : (
+        <div className="App">
+          <p className="App__message">
+            Nothing was pressed yet
+          </p>
+        </div>
+      );
```

Comment:
```suggestion
    return
        <div className="App">
          <p className="App__message">
            {pressedKey 
              ? `The last pressed key is [${pressedKey}]`
              :   Nothing was pressed yet
          </p>
        </div>
      )
```

Reply:
it's the same block and you just need to render different text so there is no need in re-rendering the whole block